TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Owner: {Your Name} {you@example.com}, {@you} on the Elm Slack
```

Comment:
Sorry to spam your inbox, Max… but I didn't say *why* "author" doesn't convey my meaning. I'll try to keep it to just this one additional comment. 😨 

Basically: an *author* is immutable. Say I put a authorship notation in a `LICENSE` file on GitHub. That authorship is de facto permanent, and someone needs my permission to change it, modify it, et cetera. Otherwise, it looks like they're putting words in my mouth or cheating by stealing someone else's code.

Other words that could work here:

- Steward
- Coordinator
- Caretaker
- Orchestrator

I like steward best, of those.